src/aead/chacha20_poly1305.rs

```diff
@@ -24,19 +24,20 @@ pub static CHACHA20_POLY1305: aead::Algorithm = aead::Algorithm {
     init: chacha20_poly1305_init,
     seal: chacha20_poly1305_seal,
     open: chacha20_poly1305_open,
+    id: aead::AlgorithmID::CHACHA20_POLY1305,
 };
 
 /// Copies |key| into |ctx_buf|.
 pub fn chacha20_poly1305_init(ctx_buf: &mut [u8], key: &[u8])
-                              -> Result<(), error::Unspecified> {
+    -> Result<(), error::Unspecified> {
```

Comment:
Probably we should reformat the code, but not in this PR. Please remove all the unrelated formatting changes.